WooCommerce/Classes/Model/CustomHelpCenterContent.swift

```diff
@@ -0,0 +1,40 @@
+import Foundation
+import WordPressAuthenticator
+
+/// For holding the custom help center content URL
+/// and analytics tracking values
+///
+struct CustomHelpCenterContent {
+    /// Custom help center web page's URL
+    ///
+    let helpCenterContentURL: URL
```

Comment:
super nit: since its struct name already indicates that it's for the help center content, just `url` might be more concise

```suggestion
    let url: URL
```

Reply:
Done in ef54d0326feb652ff7d5d658b8b3c45de87bd1e8